src/ParseFasm.cpp

```diff
@@ -0,0 +1,508 @@
+// Copyright (C) 2020  The SymbiFlow Authors.
+//
+// Use of this source code is governed by a ISC-style
+// license that can be found in the LICENSE file or at
+// https://opensource.org/licenses/ISC
+//
+// SPDX-License-Identifier: ISC
+
+#include "FasmLexer.h"
+#include "FasmParser.h"
+#include "FasmParserVisitor.h"
+#include "antlr4-runtime.h"
+
+/// This code parses FASM and produces a lightweight binary format that
+/// is fast and simple to unpack based on the tag/length/value (TLV)
+/// format.
+///
+/// For example, for a fixed width integer, this can just be:
```

Comment:
Should mention that the integer is in host endianness?  This is typically implies, but is worth mentioning?

Reply:
Done.